### PR TITLE
WINDUP-844 - Adding information to README.txt for end users of offline

### PR DESCRIPTION
**Pull request by mareknovotny**
distribution

### CODE REVIEW

**Comment by mbriskar** [comment]
for me it is OK.

File: src/main/resources/README.txt [hunk 1]
```diff
@@ -5,5 +5,55 @@
 |__/|__/_/_/ /_/\__,_/\__,_/ .___/ 
                           /_/      
 
-Windup ${project.version} is a tool used to simplify Java application migrations.
+Windup ${project.version} is an extensible and customizable rule-based tool that helps simplify migration of Java applications.is an extensible and customizable rule-based tool that helps simplify migration of Java applications.
```
Comment (OndraZizka) [inline src/main/resources/README.txt]: A typo: 
"applications.is an extensible and customizable rule-based tool that helps simplify migration of Java applications."

**Comment by OndraZizka** [comment]
I think src/main/resources is for resources which to go the packaged jars. This should rather be in src/docs or such - IMO.

File: src/main/resources/README.txt (hunk 1 above)
Comment (mareknovotny) [inline src/main/resources/README.txt]: good catch!

**Comment by mareknovotny** [comment]
that is not too strict IMHO, if you have just distribution assembly module/submodule, but this is just change of existing file which is in others in flat distribution structure which changed complicates useless the assembly

File: src/main/resources/README.txt [hunk 2]
```diff
@@ -5,5 +5,55 @@
...
                           /_/      
 
-Windup ${project.version} is a tool used to simplify Java application migrations.
+Windup ${project.version} is an extensible and customizable rule-based tool that helps simplify migration of Java applications.
 
+Windup examines application artifacts, including project source directories and applications archives, 
+then produces an HTML report highlighting areas that need changes. 
+
+Windup can be used to migrate Java applications from previous versions of Red Hat JBoss Enterprise Application Platform 
+or from other containers, such as Oracle WebLogic Server or IBM® WebSphere® Application Server.
```
Comment (jsight) [inline src/main/resources/README.txt]: Why have the ® on WebSphere but not on WebLogic?

File: src/main/resources/README.txt [hunk 3]
```diff
@@ -5,5 +5,55 @@
...
+        
+        > bin\windup.bat --input INPUT_ARCHIVE_OR_DIRECTORY --output OUTPUT_REPORT_DIRECTORY
+            --source SOURCE_TECHNOLOGY --target TARGET_TECHNOLOGY --packages PACKAGE_1 PACKAGE_2 PACKAGE_N
+
+    Getting more detailed help on possible arguments and options for command line
+
+        For Linux:
+        $ bin/windup --help
+
+        For Windows:
```
Comment (jsight) [inline src/main/resources/README.txt]: We should probably change this to "\", even though either will work.

File: src/main/resources/README.txt [hunk 4]
```diff
@@ -5,5 +5,55 @@
...
+        > bin/windup.bat --help
```
Comment (jsight) [inline src/main/resources/README.txt]: We should probably change this to "\", even though either will work.

Comment (mareknovotny) [inline src/main/resources/README.txt]: I agree, I did a few copy&paste from windup wiki and I didn't catch this.

File: src/main/resources/README.txt (hunk 2 above)
Comment (mareknovotny) [inline src/main/resources/README.txt]: good catch, again copy and paste error, I think i will remove the (R) to not complicate it.

Comment (jsight) [inline src/main/resources/README.txt]: That seems fine to me.

**Pull request merged**